### PR TITLE
feature: Add initial support for runtime rofl transactions

### PR DESCRIPTION
**Pull request by ptrus**
Fixes: https://github.com/oasisprotocol/nexus/issues/806
There were already some ROFL apps deployed on testnet for sapphire, so until we reindex, we will miss those transactions/events for now. 

It is also possible to write a script that goes over runtime transactions and events for Sapphire and backfills the rofl events and transactions if we won't do a reindex and will still want the missed data - we can do this at a a later time.

For forntend, just displaying the transations.body and formating the events nicely will probably be good for first version.

### CODE REVIEW

File: analyzer/runtime/visitors.go [hunk 1]
```diff
@@ -121,6 +126,46 @@ func VisitCall(call *sdkTypes.Call, result *sdkTypes.CallResult, handler *CallHa
...
+	case "rofl.Update":
+		if handler.RoflUpdate != nil {
+			var body rofl.Update
+			if err := cbor.Unmarshal(call.Body, &body); err != nil {
+				return fmt.Errorf("unmarshal rofl update: %w", err)
+			}
+			if err := handler.RoflUpdate(&body); err != nil {
+				return fmt.Errorf("rofl update: %w", err)
+			}
+		}
```
Comment (Andrew7234) [inline analyzer/runtime/visitors.go:148]: Is there any relevant information in the CallResult for rofl transactions? Or is it left for a future ticket?

File: .changelog/812.feature.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+feature: Add support for runtime rofl transactions
+
+New runtime transactions:
```
Comment (Andrew7234) [inline .changelog/812.feature.md:3]: Not an ask for this PR, but we should think about adding another e2e regression suite that encompasses rofl/sapphire/consensusaccount txs and the rest of the new features being added :)

**Review by Andrew7234 — APPROVED** [review]
Nice, the Explorer folks might be interested in designing iconography for rofl transactions. cc @csillag

**Comment by ptrus** [comment]
> Nice, the Explorer folks might be interested in designing iconography for rofl transactions

Already in progress https://github.com/oasisprotocol/explorer/issues/1641

File: .changelog/812.feature.md (hunk 2 above)
Comment (ptrus) [reply .changelog/812.feature.md:3]: Good idea, opened https://github.com/oasisprotocol/nexus/issues/826

File: analyzer/runtime/visitors.go (hunk 1 above)
Comment (ptrus) [reply analyzer/runtime/visitors.go:148]: "rofl.Create" has App ID, others are empty. But we don't have anything to do with it at the moment, will leave it as a followup https://github.com/oasisprotocol/nexus/issues/807

**Pull request merged**